py/kubeflow/testing/cleanup_ci.py

```diff
@@ -904,7 +906,19 @@ def cleanup_clusters(args):
       insert_time_utc = insert_time + datetime.timedelta(hours=-1 * hours_offset)
       age = datetime.datetime.utcnow()- insert_time_utc
 
-      if age > datetime.timedelta(hours=args.max_age_hours):
+      if c.get("status", "") == "ERROR":
```

Comment:
should we also GC `degraded` clusters?
https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters#Cluster.Status

Reply:
Done.